src/flashbar/flash.tsx

```diff
@@ -72,7 +72,7 @@ export const focusFlashById = throttle(
   { trailing: false }
 );
 
-export interface FlashProps extends FlashbarProps.MessageDefinition {
+export interface FlashProps extends Omit<FlashbarProps.MessageDefinition, 'testId'> {
```

Comment:
There's a bit of an inconsistency when it comes to `FlashWrapper`.

`FlashWrapper` should point to the list item and not the flash component itself:
https://github.com/cloudscape-design/components/blob/672e7f80aeeac9b1907e1a3ae13a3e702b6b5951/src/test-utils/dom/flashbar/flash.ts#L10

But the existing `findFlashByType` returns the flash rather than the parent `<li />`:
https://github.com/cloudscape-design/components/blob/672e7f80aeeac9b1907e1a3ae13a3e702b6b5951/src/test-utils/dom/flashbar/index.ts#L29

To make sure than this mistake doesn't happen with `testId` I omitted that from the `FlashProps` so that it is not assigned by mistake in the future.